saleor/data_feeds/google_merchant.py

```diff
@@ -71,10 +70,6 @@ def item_guid(item: ProductVariant):
     return item.sku
 
 
-def item_link(item: ProductVariant, current_site):
-    return add_domain(current_site.domain, item.get_absolute_url(), not settings.DEBUG)
```

Comment:
I think it would be nice to have a way to still handle it (https://support.google.com/merchants/answer/6324416?hl=en&ref_topic=6324338)